.ci/cico_rhche_compatibility_test.sh

```diff
@@ -36,25 +79,53 @@ checkAllCreds
 echo "Installing dependencies:"
 installDependencies
 
+echo "Configuring remote:"
+git remote set-url origin https://${GITHUB_AUTH_STRING}@github.com/redhat-developer/rh-che.git
+
 #Check branch for tracking changes/Create new branch
 BRANCH="upstream-check-$CHE_VERSION"
 echo "Checking if branch $BRANCH exists."
 set +e
-git checkout "$BRANCH"
+git checkout "$BRANCH" 2>&1 >/dev/null
 return_code=$?
 set -e
-if [ $return_code -eq 0 ]; then 
-  echo "Branch $BRANCH found - rebasing."
-  git rebase origin/master "$BRANCH"
-else 
+if [ ! $return_code -eq 0 ]; then
   echo "Branch $BRANCH not found - creating new one."
+  git checkout master
   git checkout -b "$BRANCH"
-	
-  #change version of used che
+  echo "Setting new branch origin"
+  git push --set-upstream origin $BRANCH
```

Comment:
Can you explain the meaning of this line - what it was before and why to change that?

Reply:
At that point, the branch doesn't have an upstream tracking branch. This command creates it in origin, so that the PR can be created